packages/integration-tests/api/EarthdataLogin.js

```diff
@@ -70,8 +70,9 @@ async function getEarthdataAccessToken({ redirectUri, requestOrigin }) {
     );
   }
 
-  const accessTokenResponse = await earthdataLoginClient.getAccessToken(authorizationCode);
+  let accessTokenResponse = await earthdataLoginClient.getAccessToken(authorizationCode);
 
+  accessTokenResponse = { ...accessTokenResponse, ...userParams };
```

Comment:
I know I'm being dense here, but what is storing the username in the access tokens table getting for us?

Reply:
In my case I want to know that the credentials that were returned from the lambda belong to the userName who asked for them.  
So when I make a fake factory here: https://github.com/nasa/cumulus/blob/CUMULUS-1199-move-s3credential-endpoint/example/spec/s3Access/s3AccessSpec.js#L121

I can look at the information from the STS.getCallerIdentity() and see that the credentials are associated with that username: https://github.com/nasa/cumulus/blob/CUMULUS-1199-move-s3credential-endpoint/example/spec/s3Access/s3AccessSpec.js#L121

Reply:
Ah, I see where the username gets pulled out of the access token record by the s3 credentials endpoint: https://github.com/nasa/cumulus/pull/881/files#diff-a839d1b4978cafe0031568b43ce88e1bR138

Reply:
Yup, that's how we know who's calling the endpoint.  But the test, is validating that that value was called through to the endpoint, and that the endpoints `accessKeyId`, `secretAccessKey`, and `sessionToken`, have that user it the STS caller identity.